templates/room.hbs

```diff
@@ -86,12 +86,10 @@
                             <a-entity
                                 networked-audio-source
                                 networked-audio-analyser
-                                matcolor-audio-feedback="objectName: Head_Mesh"
```

Comment:
Did you mean to remove this? I think you just want to get rid of `objectName` since you changed it to target the mesh of the entity it's on.

Reply:
`.Head` is actually a bone so I can't use `matcolor-audio-feedback`. We need to find a better way to do this.